gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,368 @@
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+Attempting to run Flex Gateway with an expired certificate in Local Mode results in the following:
+
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+If a certificate expires, you can still invoke `flexctl registration renew`, which updates the expired certificate.
+
+To avoid disruptions:
+
+. <<before-you-begin,Download the latest version of Flex Gateway.>>
+. Renew your Flex Gateway registration.
++
+Choose one of two methods:
++
+* <<renew-with-user-password,Anypoint Platform User Credentials>>
+* <<renew-with-conn-app,Connected App Credentials>>
+
+. Verify the success of the renewal process.
++
+Choose one of two methods:
++
+* <<inspect-with-custom-url,Admin API URL>>
+* <<inspect-with-reg-file,Registration File>>
+
+[[before-you-begin]]
+== Before You Begin
+
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. You don't need to register and run this version of the gateway.
+
+* xref:flex-install.adoc[]
+
+[[renew-with-user-password]]
+== Renew Registration with Anypoint Platform User Credentials
+
+. Create a new directory called `flex-renew-registration` (or similar). You run the `renew` command in this new directory.
+
+. Use the following command:
++
+[tabs]
+====
+Linux::
++
+[source,ssh]
+----
+flexctl registration renew --username=<your-username> --password=<your-password> \
+ <path-to-registration-file>
+----
++
+Use `sudo` if you encounter file permission issues when running this command.
+
+Docker::
++
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command.
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v "$(pwd)":/renew \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+registration renew \
+--username=<your-username> \
+--password=<your-password> \
+--output-directory=/renew \
+<path-to-registration-file>
+----
+
+Podman::
++
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command.
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v "$(pwd)":/renew:Z \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+registration renew \
+--username=<your-username> \
+--password=<your-password> \
+--output-directory=/renew \
+<path-to-registration-file>
+----
+====
+
+. Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. You can optionally output the new registration file to a directory by using the `--output-directory` flag. The `renew` command overwrites the existing registration file if you specify the file's directory as the output directory.
```

Comment:
a small change, the renew command cannot overwrite an existing registration

Reply:
Fixed!